src/sentry/web/frontend/error_page_embed.py

```diff
@@ -154,7 +157,7 @@ def dispatch(self, request):
 
             try:
                 event = Event.objects.filter(project_id=report.project.id,
-                                             event_id=report.event_id).select_related('group')[0]
```

Comment:
Was the `select_related` just not needed?

Reply:
This relationship doesn't exist. So it did nothing.